### PR TITLE
Parse RDATE property

### PR DESCRIPTION
**Pull request by romarickb**
Parse `RDATE` property

For more information, read https://www.kanzaki.com/docs/ical/rdate.html

```
The property is defined by the following notation:

  rdate      = "RDATE" rdtparam ":" rdtval *("," rdtval) CRLF

  rdtparam   = *(

             ; the following are optional,
             ; but MUST NOT occur more than once

             (";" "VALUE" "=" ("DATE-TIME" / "DATE" / "PERIOD")) /
             (";" tzidparam) /

             ; the following is optional,
             ; and MAY occur more than once

             (";" xparam)

             )

  rdtval     = date-time / date / period
  ;Value MUST match value type
```

### CODE REVIEW

File: lib/ex_ical/event.ex [hunk 1]
```diff
@@ -33,6 +33,10 @@ defmodule ExIcal.Event do
       Defines a rule or repeating pattern for recurring events. Corresponds to
       the iCal `RRULE` property ([4.8.5.4 Recurrence Rule]).
 
+    - `rdate`:
+      Defines a a of date/times for a recurrence set. Corresponds to
```
Comment (garno) [inline lib/ex_ical/event.ex]: Not sure I understand this sentence :)

File: test/ex_ical/date_parser_test.exs [hunk 2]
```diff
@@ -16,7 +16,7 @@ defmodule ExIcal.DateParserTest do
     # Input Datestring |   Parsed Date |  Timezone When Global TZID = ?   |
     #                  |               |           nil |  America/Chicago |
     #------------------+--------------------------------------------------+
-    {        "19690620",     date_match,  utc_tzmatch,      utc_tzmatch,},
+    {        "19690620",     date_match,  utc_tzmatch,  chicago_tzmatch,},
```
Comment (garno) [inline test/ex_ical/date_parser_test.exs:19]: Why this change?

**Review by garno — APPROVED** [review]
I’m not really ical-savvy but the tests look fine 👌

File: test/ex_ical/date_parser_test.exs (hunk 2 above)
Comment (romarickb) [reply test/ex_ical/date_parser_test.exs:19]: In my opinion if we have timezone information, we should use it. 19690620 should be at `00:00:00` chicago time.
Do you think that's a mistake to considère timezone information in this case ?

Comment (garno) [reply test/ex_ical/date_parser_test.exs:19]: No I think it’s ok, I was wondering why the test data changed when we simply taken into account the `RDATE` property.
